source/MRMesh/MRMeshBuilder.cpp

```diff
@@ -162,10 +162,14 @@ AddFaceResult FaceAdder::add( MeshTopology & m, FaceId face, const VertId * firs
     }
 
     // create missing face edges
+    int createdEdgeCount = 0;
     for ( int i = 0; i < sz; ++i )
     {
         if ( !e_[i].valid() )
+        {
             e_[i] = m.makeEdge();
+            ++createdEdgeCount;
+        }
```

Comment:
why do we need this? looks like it is not used

Reply:
old code. removed